src/sage/matroids/oriented_matroids/circuit_oriented_matroid.py

```diff
@@ -0,0 +1,263 @@
+r"""
+Oriented matroid with circuit axioms
+
+This implements an oriented matroid using the circuit axioms.
+
+AUTHORS:
+
+- Aram Dermenjian (2019-07-12): Initial version
+"""
+
+# ****************************************************************************
+#      Copyright (C) 2019   Aram Dermenjian <aram.dermenjian.math at gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#                  https://www.gnu.org/licenses/
+# ****************************************************************************
+
+from __future__ import annotations
```

Comment:
```suggestion
```
We are in the future already!

Reply:
I needed this for the `is_valid` functions. Without this, it wasn't handling the `bool | tuple[bool, dict]` correctly. The `|` (pipe) was added in 3.10 and since Sage uses (potentially) versions before 3.10, we do actually need this from the future =)